src/Chapter2.hs

```diff
@@ -741,7 +780,11 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: (Eq a) => a -> [[a]] -> [[a]]
+contains = go
+  where
+    go _ []       = []
+    go x (xs:xss) = if elem x xs then xs : go x xss else go x xss
```

Comment:
You can use `filter` function instead of manual walk on the list. `filter` also handles the empty lists, so you could skip this case 👌🏼

Reply:
I will update my answer !